projects/igniteui-angular/src/lib/directives/mask/mask.directive.ts

```diff
@@ -47,6 +48,34 @@ export class IgxMaskDirective implements OnInit, ControlValueAccessor {
      */
     @Input()
     public includeLiterals: boolean;
+    /**
+     * Specifies a placeholder.
+     * ```html
+     * <input placeholder = "emty">
+     * ```
+     * @memberof IgxMaskDirective
+     */
+    @Input()
+    public placeholder: string;
+    /**
+     * Specifies a pipe to be used on blur.
+     * ```html
+     * <input [displayValuePipe] = "displayFormatPipe">
+     * ```
+     * @memberof IgxMaskDirective
+     */
+    @Input()
+    public displayValuePipe: PipeTransform;
+    /**
+     * Specifies a pipe to be used on focus.
+     * ```html
+     * <input [inputValuePipe] = "inputFormatPipe">
+     * ```
+     * @memberof IgxMaskDirective
+     */
+    @Input()
+    public inputValuePipe: PipeTransform;
```

Comment:
The name of the property and the comment are misleading. The comment says the pipe is used only when the control is focused, but the name of the property implies that it would be used when the user inputs value.